database/migrations/postgres/1753262713869421_separate_deployment_attempts_history.up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS deployment_attempts_history
```

Comment:
code changes are fine, I'm only holding back on the approval until we aligned with @miguel-crespo-fdc  about the varchar.